### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by x183**
prisma db push should be recognised in the readme

### CODE REVIEW

**Review by GAsplund — COMMENTED** [review]
All good, check suggestion.

**Review by GAsplund — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -16,7 +16,11 @@ Then you need to install pnpm by running `npm install -g pnpm`
 
 ## Starting the development database
 
+Now you need to create the database schema, you do this by, from the root of the project, running `cd backend & pnpm prisma db push`
+
 After this you need to start the database, you do this by running `docker compose -f docker-compose.dev.yml up -d` in the source-folder of the project.
```
Comment (Supergamer1337) [inline README.md:21]: For `prisma db push` the database already needs to be up and running...

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]

**Review by Supergamer1337 — APPROVED** [review]
Looks good now!

**Review by GAsplund — APPROVED** [review]

**Pull request merged**